packages/app-store/closecomothercalendar/pages/setup/index.tsx

```diff
@@ -0,0 +1,164 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
+import { Check, X } from "react-feather";
+import { useForm, Controller } from "react-hook-form";
+import { Toaster } from "react-hot-toast";
+import z from "zod";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import showToast from "@calcom/lib/notification";
+import Button from "@calcom/ui/v2/Button";
+import { Form, TextField } from "@calcom/ui/v2/form/fields";
+
+export interface ICloseComSetupProps {
+  apiKey: string;
+}
+
+const formSchema = z.object({
+  api_key: z.string(),
+});
+
+export default function CloseComSetup(props: ICloseComSetupProps) {
```

Comment:
This setup page is to gather the API key from the user, as the API works with Basic Auth. It features a "Test API Key" button to simply check if the key is valid or not.